code/modules/crafting/recipes.dm

```diff
@@ -439,6 +439,13 @@
 	result = /obj/structure/bonfire
 	category = CAT_PRIMAL
 
+/datum/crafting_recipe/brewbarrel
+	name = "Brewing barrel"
+	time = 60
+	reqs = list(/obj/item/weapon/grown/log = 5)
```

Comment:
Building things from logs is kind of a special pain compared to using wood planks. Please consider all options here.